core/arch/arm/kernel/thread_spmc.c

```diff
@@ -810,6 +811,19 @@ static void handle_mem_reclaim(struct thread_smc_args *args)
 	set_args(args, ret_fid, ret_val, 0, 0, 0, 0);
 }
 
+void __spmc_sp_msg_handler(struct thread_smc_args *args,
+			   struct sp_session *caller_sp);
+
+void __spmc_sp_msg_handler(struct thread_smc_args *args __maybe_unused,
+			   struct sp_session *caller_sp __maybe_unused)
+{
+#ifdef CFG_SECURE_PARTITION
```

Comment:
I'd rather ifdef around `spmc_sp_thread_entry()` in `core/arch/arm/kernel/thread_spmc_a64.S` than this.

Reply:
Done